tests/plotting/test_actor.py

```diff
@@ -218,9 +218,20 @@ def test_actor_length(klass, actor, dummy_actor):
     initial_length = 2**0.5  # sqrt(2)
     scale_factor = 2
 
-    assert actor.length == initial_length
+    assert np.allclose(actor.length, initial_length)
```

Comment:
Scalar comparison (note: this change is not related to the new size parameter)
```suggestion
    assert np.isclose(actor.length, initial_length)
```